shared/src/main/java/uk/gov/di/authentication/shared/helpers/ValidationHelper.java

```diff
@@ -84,6 +86,10 @@ public static Optional<ErrorResponse> validatePhoneNumber(
         }
     }
 
+    static boolean isAcceptedPhoneNumberType(PhoneNumberType phoneNumberType) {
+        return MOBILE.equals(phoneNumberType) || FIXED_LINE_OR_MOBILE.equals(phoneNumberType);
+    }
+
```

Comment:
I wonder if we should log out here the phone number type which might help identifying any issues with users not receiving SMS?

Reply:
have added